R/getPrevalence.R

```diff
@@ -517,7 +517,17 @@ setMethod("getPrevalentAbundance", signature = c(x = "SummarizedExperiment"),
 #'
 #' @param other_label A single \code{character} valued used as the label for the
 #'   summary of non-prevalent taxa. (default: \code{other_label = "Other"})
-#'
+#'   
+#' @param agglomerate.tree \code{TRUE} or \code{FALSE}: should
+#'   \code{rowTree()} also be agglomerated? (Default:
+#'   \code{agglomerate.tree = FALSE})
+#'
+#' @param mergeRefSeq logical scalar: Should a consensus sequence be calculated?
+#' If set to \code{FALSE}, the result
+#' from \code{archetype} is returned; If set to \code{TRUE} the result
+#' from \code{\link[DECIPHER:ConsensusSequence]{DECIPHER::ConsensusSequence}}
+#' is returned. (Default: \code{mergeRefSeq = FALSE})
```

Comment:
These will be changed by this PR https://github.com/microbiome/mia/pull/582 which takes the priority